lib/ios/native-navigation/Dictionary+FunctionalExtensions.swift

```diff
@@ -40,12 +40,14 @@ extension Dictionary {
     return dict
   }
 
-
+#if swift(>=3.2)
```

Comment:
@henrikra here's the answer https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Compatibility.html